bin/apisix

```diff
@@ -181,7 +181,10 @@ http {
     lua_shared_dict worker-events        10m;
     lua_shared_dict lrucache-lock        10m;
     lua_shared_dict skywalking-tracing-buffer    100m;
-
+    lua_shared_dict balancer_ewma 10m;
+    lua_shared_dict balancer_ewma_last_touched_at 10m;
+    lua_shared_dict balancer_ewma_locks 10m;
```

Comment:
need to align

![image](https://user-images.githubusercontent.com/6814606/89438009-dbb0cb00-d77a-11ea-93ca-316bead5ddad.png)